docs/content/config/security/mail_crypt.md

```diff
@@ -0,0 +1,54 @@
+---
+title: 'Security | mail_crypt (email/storage encryption)'
+---
+
+Official Dovecot documentation: https://doc.dovecot.org/configuration_manual/mail_crypt_plugin/
+
+> The Mail crypt plugin is used to secure email messages stored in a Dovecot system. Messages are encrypted before written to storage and decrypted after reading. Both operations are transparent to the user.
+>
+> In case of unauthorized access to the storage backend, the messages will, without access to the decryption keys, be unreadable to the offending party.
+>
+> There can be a single encryption key for the whole system or each user can have a key of their own. The used cryptographical methods are widely used standards and keys are stored in portable formats, when possible.
```

Comment:
You could use the `!!! info` admonitions stlye here. [See here for example](https://docker-mailserver.github.io/docker-mailserver/edge/config/setup.sh/)

Reply:
Perfect <3